### PR TITLE
Add a copy of config.json files in the datadir

### PR DESCRIPTION
**Pull request by landryb**
This way, administrators can directly configure/override the
addons they want to use.

### CODE REVIEW

**Comment by landryb** [comment]
Only drawback with this PR is that there's a new copy of the file in the datadir, which has to be kept in sync with the one in the webapp if the latter changes... so to be debated :)

**Comment by landryb** [comment]
Or we could reuse the fact that when building a package, the maven vodoo checkouts the datadir and dispatches the config.json files in the webapp addons subdirs ? @pmauduit do you think it's possible, to avoid duplicating the files ?

File: mapfishapp/addons/fullscreen/config.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[{
+    "id": "fullscreen_0",
+    "enabled": false,
```
Comment (fvanderbiest) [inline mapfishapp/addons/fullscreen/config.json]: true

File: mapfishapp/addons/fullscreen/config.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "name": "Fullscreen",
+    "title": {
+        "en": "Fullscreen",
+        "es": "Pantalla Completa",
+        "fr": "Plein écran",
+        "de": "Vollbild"
+    },
+    "options": {
+        "toolbars": true
```
Comment (fvanderbiest) [inline mapfishapp/addons/fullscreen/config.json]: false is nicer by default IMO

File: mapfishapp/addons/locateme/config.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+[{
+    "id": "locateme_0",
+    "name": "LocateMe",
+    "enabled": false,
```
Comment (fvanderbiest) [inline mapfishapp/addons/locateme/config.json]: true

File: mapfishapp/addons/measure/config.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+[{
+    "id": "measure_0",
+    "name": "Measure",
+    "enabled": false,
```
Comment (fvanderbiest) [inline mapfishapp/addons/measure/config.json]: true

File: mapfishapp/addons/osm2geor/config.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
+[{
+  "id": "osm2geor_0",
+  "name": "Osm2Geor",
+  "enabled": false,
```
Comment (fvanderbiest) [inline mapfishapp/addons/osm2geor/config.json]: true

**Comment by fvanderbiest** [comment]
LGTM, thanks

**Pull request merged**